egs/vaevc/vcc2018v1/conf/default.yml

```diff
@@ -0,0 +1,130 @@
+# feature setting
```

Comment:
Is it ok to make default.yml as symbolic link from ```template/conf```? Maybe you need to overwrite it by ```vcc2020v1/conf/default.yml```.

Reply:
Sorry, I don't really understand... the current design is to put separate configs for each dataset, right?

Reply:
Sorry, it requires to change feature extraction settings in each recipe.
I thought the word of default should be default for vaevc not each recipe, but it's my misunderstandings.

Reply:
I see. Or shall we use more meaningful names? For example, if we want to compare different network types or features, we can name it to something like `mlfb.yml`, `dilatedcnn.yml`?

Reply:
That's good idea. ```${feat_type}_${trainer_type}.yml``` like ```{mlfb}_{vqvae}.yml``` is fine.